gorm-tools/src/main/groovy/gorm/tools/job/SyncJobArgs.groovy

```diff
@@ -54,6 +54,13 @@ class SyncJobArgs {
      */
     Boolean saveDataAsFile = false
 
+    /**
+     * If errors should be stored separate from result data.
+     * If true, syncjob.errorBytes will be populated with error results
+     * XXX: Currently it doesnt work with `saveDataAsFile`. saves errors to `errorBytes` field of `SyncJob`.
+     */
+    Boolean saveErrorsSeparate = false
```

Comment:
see note to change name, also, for this one we should never save to file and always go to db.